crates/cli-tools/src/changelog.rs

```diff
@@ -18,9 +18,11 @@ use std::fmt::{Display, Write};
 use std::io::BufRead;
 
 use anyhow::{anyhow, bail, ensure, Context, Result};
+use cargo_metadata as _;
```

Comment:
Added to provide Semver error impl.

We will need soon anyways to validate latest version in changelog file matches cargo.toml